cmd/concourse/main.go

```diff
@@ -1,4 +1,4 @@
-//go:generate go run github.com/markbates/pkger/cmd/pkger
```

Comment:
this fix (and the moving `pkged.go` under `cmd/concourse`) should be backported to 6.7.x; without it, `fly` commands shell out to `go` on every run and seem to result in fetching and building `github.com/gobuffalo/here`, which is scary

Reply:
backported: #6480

Reply:
We merged the fix into `master` in #6482 so we could unblock the `release/7.0.x` branch